src/main/java/org/jenkinsci/test/acceptance/po/TopLevelItem.java

```diff
@@ -43,12 +43,12 @@ public TopLevelItem(PageObject context, URL url, String name) {
      */
     @CheckReturnValue
     public <T extends TopLevelItem> T renameTo(final String newName) {
-        configure();
+        open();
         String oldName = name;
-        control("/name").set(newName);
-        save();
-        waitFor(by.button("Yes")).click();
-
+        control(by.href("/job/" + oldName + "/confirm-rename")).click();
```

Comment:
Does this link exist in Jenkins LTS?

Reply:
Good catch! I will try to make this work in both cases